ceremonies/src/tests.rs

```diff
@@ -3153,10 +3153,25 @@ fn has_reputation_works() {
 
 		assert_eq!(EncointerCeremonies::has_reputation(&alice, &cid), false);
 
+		// acausal cindex
 		EncointerCeremonies::fake_reputation((cid, 4), &alice, Reputation::VerifiedUnlinked);
 
 		assert_eq!(EncointerCeremonies::has_reputation(&alice, &cid), false);
 
+		// reputation type does not qualify
+		EncointerCeremonies::fake_reputation((cid, 1), &alice, Reputation::Unverified);
+
+		assert_eq!(EncointerCeremonies::has_reputation(&alice, &cid), false);
+
+		EncointerCeremonies::fake_reputation((cid, 1), &alice, Reputation::UnverifiedReputable);
+
+		assert_eq!(EncointerCeremonies::has_reputation(&alice, &cid), false);
```

Comment:
I actually wonder if we shouldn't accept `UnverifiedReputable` as `has_reputation`, because the account is just an alias and the controller of the account has reputation (even if rep is on another account.

@pifragile ?

Reply:
I understand your reasoning but I'm not sure, as like this reputations would be counted twice. You can have reputation with account A, then use this reputation to register account B for a gathering. B then becomes `UnverifiedReputable`. Then both accounts A and B could endorse newbies.

Reply:
True. No harm done, but still it may make sense to leave it as is